enrol/autoenrol/lib.php

```diff
@@ -54,7 +56,7 @@ public function roles_protected() {
      * @return bool
      */
     public function allow_unenrol(stdClass $instance) {
-        // Users with unenrol cap may unenrol other users manually - requires enrol/autoenrol:unenrol.
+        // Users with onenrol cap may unenrol other users manually - requires enrol/autoenrol:unenrol.
```

Comment:
Hi @bobopinna 

Was this change deliberate? I'm just wondering whether this really does mean "on enrol" now?